l10n_it_withholding_tax_payment/i18n/l10n_it_withholding_tax_payment.pot

```diff
@@ -0,0 +1,268 @@
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__write_uid
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__write_uid
+msgid "Last Updated by"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__write_date
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__write_date
+msgid "Last Updated on"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__line_ids
+msgid "Lines"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move__wt_move_payment_id
+msgid "Move Payment"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__name
+msgid "Name"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+msgid "Period"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+msgid "Set to Draft"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__state
+msgid "Status"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: code:addons/l10n_it_withholding_tax_payment/models/withholding_tax.py:131
+#, python-format
+msgid "The selected moves must have the same company!"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+msgid "Tot"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.ui.menu,name:l10n_it_withholding_tax_payment.menu_withholding_tax_move_payment
+msgid "WT Move Payments"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__wt_move_ids
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.wizard_wt_move_payment_create_view
+msgid "WT Moves"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__amount
+msgid "WT amount"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model,name:l10n_it_withholding_tax_payment.model_wizard_wt_move_payment_create
+msgid "WT wizard create move payment"
```

Comment:
Non mi è molto chiara questa stringa.

Reply:
Neanche a me, a dirla tutta...  
In ogni caso, la ignorerò bellamente: non verrà mai mostrata a video a nessun utente in alcun caso.

È solo una descrizione di un modello temporaneo; è sempre buona norma inserire qualcosa, onde evitare dei `warning` superflui all'interno dei log di Odoo.  
In questo caso, non è altro che il nome del modello stesso.

Reply:
> Neanche a me, a dirla tutta...
> In ogni caso, la ignorerò bellamente: non verrà mai mostrata a video a nessun utente in alcun caso.

Eh, facile. Il problema poi se lo trova chi dovrà tradurlo però. :laughing: 
Suggerisco di mettere "Create WT Payment Move Wizard".

Reply:
In maniera molto semplice, perché non ha ragione di essere tradotta! 🙃

Un altro esempio, di stringa non traducibile ma sempre presente in un `.pot`, può essere:
https://github.com/OCA/l10n-italy/blob/a3ca0dd74703000b8002450ae9e1389811ab148b/l10n_it_fatturapa_out/i18n/l10n_it_fatturapa_out.pot#L229

---

L'uso della `_description` (con la relativa traduzione) avviene in 2 singoli casi:

- Durante la visualizzazione di eventuali messaggi di errore dovuti a mancati permessi di scrittura per l'utente corrente del modello che si sta tentando di modificare.
- Durante la creazione di un nuovo record di un modello con Chatter; viene inserito automaticamente un messaggio dal formato poco differente da: "Nuovo <_description> creato."

Entrambi questi casi, però, non si mappano ai `TransientModel` e ai conseguenti wizard.

Ma, dato il suggerimento... Corretto! 👌

Reply:
Ottimo! Grazie della spiegazione per _description. :-)